railties/lib/rails/generators/rails/credentials/credentials_generator.rb

```diff
@@ -34,8 +34,6 @@ def add_credentials_file_silently(template = nil)
       private
         def credentials
           ActiveSupport::EncryptedConfiguration.new(
-            config_path: "config/credentials.yml.enc",
```

Comment:
Encrypted Configuration isn’t just for credentials as you can see here. So the extraction is wrong.